core/blockchain.go

```diff
@@ -320,10 +320,9 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, genesis *Genesis
 		// but that's left out to a later PR since there's not really a need
 		// right now.
 		bc.stateCache.InitTransitionStatus(true, true)
-		bc.stateCache.EndVerkleTransition()
```

Comment:
This was odd, and also now not required. `InitTransitionStatus(true, true)` already signals the important action, at least for replay.

This `EndVerkleTransition()` was odd since it was forced upfront, but in the replay you might get a `StartVerkleTransition(...)` after, which isn't coherent.